staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go

```diff
@@ -280,6 +288,10 @@ func (s *WatchServer) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 			if len(ch) == 0 {
 				flusher.Flush()
 			}
+			if isWatchListLatencyRecordingRequired {
+				recordWatchListLatency(req.Context(), s.Scope)
+				isWatchListLatencyRecordingRequired = false
```

Comment:
I might be missing something, but this doesn't seem to have any effect does it?

Reply:
oh, you are right, I can remove it now. thanks!